### DISCUSSION TITLE
Search doesn't work

### DISCUSSION THREAD

**Discussion by Choochoote** [dotnet/docfx#9936] [Q&A] [answered]
Hi, I've built a website then hosted it on github pages:
https://choochoote.github.io/TopSolidDeveloperCenter/

Problem is that I am not able to search, it's spinning around and never ending.

I've tried to add a web.config file but it didn't help.

Any idea what I can do to fix this?

**Comment by filzrev** [comment, marked as answer]
It seems `Search Index File` (`index.json`) is not deployed to GitHub Pages root dir.
`https://choochoote.github.io/TopSolidDeveloperCenter/index.json`

And above site using older docfx version `2.59.4` (Last version that running on `.NET Framework`)
If no special requirement. It's recommended to use latest version of docfx.

**Comment by Choochoote** [comment]
Thanks for your Kind help. I created index.json by adding necessary line into my docfx.json file. I’ m stuck With this version right now but plan to upgrade asap.

**Comment by filzrev** [comment]
@Choochoote
Search problems seem to be solved by adding an `index.json` file.
Is it able to close this issue if there are no further questions?
